### ISSUE TITLE
Feature requests for new components

### ISSUE THREAD

**Issue by addyosmani** [google/material-design-lite#92]
There are probably components you would like to see implemented in MDL that we don't yet provide. This is a centi-thread to capture those requests. We can't guarantee we'll be able to implement them all, but we will definitely review this thread at regular intervals.

Please note when requesting a component that we are often gated on components or styles having a presence in the [Material Design specification](https://www.google.com/design/spec/material-design/introduction.html) and may need to wait for input from our designers before we can proceed here.

If the component you're requesting isn't in the spec, we will try bringing it to the attention of the Material Design team based on demand and will consider implementing the component if guidance for it does land in the spec.

**Comment by LCallaghan84** [comment]
I have 1 minor suggestion/addition to the cards, which is Card Reveal

example/ref - http://materializecss.com/cards.html#reveal

would it be possible to add?

**Comment by Garbee** [comment]
@LCallaghan84 It is not specified by Material Design. On something like that, we would need to talk with the MD team to see if it fits into the specification somehow and how it should be done.

You however are free to do this on your own now with custom code. Nothing is stopping that.

**Comment by Garbee** [comment]
Let's actually kill this old issue. Requests for new components can follow the README's instructions of filing a [new issue](https://github.com/Google/material-design-lite/issues/new?title=%5BComponent%20Request%5D%20%7BComponent%7D&body=Please%20include:%0A*%20Description%0A*%20Material%20Design%20Spec%20link%0A*%20Use%20Case%28s%29) and providing the information asked for in the template. We can easily track requests made this way. Keeping this issue open will become too noisy to be effective for tracking.

**Comment by LCallaghan84** [comment]
I am certain it would not affect the rest of the material design

have a look here, http://countdown.teamfreewill.net/mobile/ 
using materialize but hoping to switch to MDL soon

**Comment by Garbee** [comment]
Whether it would affect other things isn't an issue. It is about the philosophy of Material Design itself. Hiding content like this, and using the menu button to trigger a non-menu item both seem against MD to me. We can bring it up with the team though and see what they think. If they say it is good, then it can get added in with a future release.

We simply don't add things because they seem alright. We verify with the Material Design team that what we implement upholds their ideology and philosophy of design.

**Comment by LCallaghan84** [comment]
What would be the alternative to the card reveal/overlay ?

**Comment by Garbee** [comment]
Support questions should be directed to [StackOverflow](https://stackoverflow.com/questions/tagged/material-design-lite) using the `material-design-lite` tag. This way the content can easily be found by others needing help.

**Comment by LCallaghan84** [comment]
Alright, will do. I must say MDL is looking and working beautifully, learning the framework right now. well done :+1:

**Comment by also** [comment]
@Garbee The [FAQ](https://github.com/google/material-design-lite/blob/master/docs/_pages/faq.md#can-i-request-or-contribute-components-to-mdl) directs people to this thread to request new components. Should it point them to the README instead?

**Comment by Garbee** [comment]
@also Thanks for pointing that out. I must have not realized when reviewing. I'll submit a fix to the FAQ and see what the rest of the team says with that.

**Comment by Garbee** [comment]
@also Fix should go out with the next production deploy. Thanks for letting us know!

**Comment by innerpeace0** [comment]
Styles and sizes of fonts for <p>, button, menu, popups, and hints for inputs aren't listed on typography page, even though they may be included in the css.

**Comment by Garbee** [comment]
@innerpeace0 Please file a new issue for that discussion. It is unrelated to this issue.

Since this issue should not be used anymore, I'm going to go ahead and lock it. If somewhere is pointing you here to request a new component instead of the [Issue template](https://github.com/Google/material-design-lite/issues/new?title=%5BComponent%20Request%5D%20%7BComponent%7D&body=Please%20include:%0A*%20Description%0A*%20Material%20Design%20Spec%20link%0A*%20Use%20Case%28s%29), please file an issue to let us know where it is so it can be updated. Thank you!
